bolt.php

```diff
@@ -18,6 +18,34 @@
     return parse('{{release_path}}/bin/console --no-interaction');
 });
 
+set('bin/php', function () {
+    $bin = get('bin', []);
+    return isset($bin['php']) ? $bin['php'] : locateBinaryPath('php');
+});
+
+set('bin/git', function () {
+    $bin = get('bin', []);
+    return isset($bin['git']) ? $bin['git'] : locateBinaryPath('git');
+});
+
+set('bin/composer', function () {
+    $bin = get('bin', []);
+    if (isset($bin['composer'])) {
+        return $bin['composer'];
+    }
+
+    if (commandExist('composer')) {
+        $composer = locateBinaryPath('composer');
+    }
+
+    if (empty($composer)) {
+        run("cd {{release_path}} && curl -sS https://getcomposer.org/installer | {{bin/php}}");
```

Comment:
Can we add some output here? Would be nice to let the user know we didn't find a composer binary, hence we're fetching one now.

Reply:
This bit is taken straight from Deployer itself. You can't call `__parent()` or somesuch, so i've copied the method and added this bit to allow to override it: 

```
    $bin = get('bin', []);
    if (isset($bin['composer'])) {
        return $bin['composer'];
    }
```

I don't think we should mess with the original parts.

Reply:
Ah, in that case, sure! 👍